gamestonk_terminal/portfolio/portfolio_analysis/pa_controller.py

```diff
@@ -96,19 +115,150 @@ def call_quit(self, _):
 
     def call_load(self, other_args):
         """Process csv command"""
-        self.portfolio_name, self.portfolio = portfolio_parser.load_csv_portfolio(
-            other_args
+        parser = argparse.ArgumentParser(
+            prog="load",
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            description="Function to get portfolio from predefined csv file inside portfolios folder",
+        )
+        parser.add_argument(
+            "-s" "--sector",
+            action="store_true",
+            default=False,
+            help="Add sector to dataframe",
+            dest="sector",
+        )
+        parser.add_argument(
+            "--no_last_price",
+            action="store_false",
+            default=True,
+            help="Dont add last price from yfinance",
+            dest="last_price",
+        )
+        parser.add_argument(
```

Comment:
Is this arg really necessary?

Reply:
Really necessary -- no.  But it can save time if someone has quite a few columns and is trying to do something that doesn't require the last price - such as looking at dividends or cost basis etc.

Reply:
But in what case would we want "nan" ?

Reply:
I thought you meant the last price argument lol.

The case for having nan I can think of is lets say you have SPY.  When you do the yfinance call, it will have nan for all some of its information (think because its an etf). So if I wanted to pull dividends for example, stocks ABC would have them and SPY wouldn't.  This flag would make sure that dividend isn't dropped.